lib/pedant_mysql2.rb

```diff
@@ -46,7 +46,11 @@ def whitelist
     end
 
     def ignored?(warning)
-      whitelist.any? { |matcher| matcher =~ warning.message }
+      whitelist.any? { |matcher| matcher =~ warning.message } || drop_table_warning(warning)
+    end
+
+    def drop_table_warning(warning)
+      warning.query =~ /\ADROP TABLE IF EXISTS/ || warning.message =~ /\AUnknown table/
```

Comment:
@Edouard-chin 👋  what was the reason for deciding to do this gem-wide rather than in a project's own `PedantMysql2.ignore(...)`? Seems now pedant is deciding what it considers a warning, rather than MySQL.